service/src/main/java/org/jboss/aerogear/unifiedpush/service/impl/ClientInstallationServiceImpl.java

```diff
@@ -83,8 +83,9 @@ public void addInstallation(Variant variant, Installation entity) {
 
     private void mergeCategories(Installation entity) {
```

Comment:
We could move the comment that is inside the method ` //replace json deserialised categories with their persistent counter parts see Category.equals` to the javadoc level, no ?